content/posts/March-2021.md

```diff
@@ -33,3 +33,16 @@ weight: 77
   created. ([packit-service#1019](https://github.com/packit/packit-service/pull/1019))
 - Matěj fixed an ugly bug which caused Packit Service not to react to
   installation events. ([packit-service#1018](https://github.com/packit/packit-service/pull/1018))
+
+## Week 11 (March 15th - March 19th)
+
+- Laura fixed a bug in processing of GitHub installations.
+  ([packit-service#1020](https://github.com/packit/packit-service/pull/1020))
+- MatejM added `fmf_url` and `fmf_ref` configuration options for testing farm,
```

Comment:
Needs also (yet unmerged) https://github.com/packit/packit-service/pull/1021 so I'm not sure whether to mention it now.